62.CAD/vertex_shader.hlsl

```diff
@@ -250,10 +250,13 @@ PSInput main(uint vertexID : SV_VertexID)
             double2 ndc = mul(transformation, double3(points[i], 1)).xy; // Transform to NDC
             transformedPoints[i] = (float2)((ndc + 1.0) * 0.5 * globals.resolution); // Transform to Screen Space
         }
+        
+        nbl::hlsl::shapes::QuadraticBezier<float> quadraticBezier = nbl::hlsl::shapes::QuadraticBezier<float>::construct(transformedPoints[0u], transformedPoints[1u], transformedPoints[2u]);
+        nbl::hlsl::shapes::Quadratic<float> quadratic = nbl::hlsl::shapes::Quadratic<float>::constructFromBezier(quadraticBezier);
+        QuadBezierAnalyticArcLengthCalculator<float> preCompData = QuadBezierAnalyticArcLengthCalculator<float>::construct(transformedPoints[0u], transformedPoints[1u], transformedPoints[2u]);
```

Comment:
this `QuadBezierAnalyticArcLengthCalculator` will change, also instead of passing the transformed points you'll be passing `quadratic` directly (changess to the struct was disclosed before)

Reply:
done